docs/CONFIG.md

```diff
@@ -5035,6 +5075,17 @@ Depending on `SelectionMode`, this represents a difference in the number of bloc
 
 Set to 0 to disable this check.
 
+### LeaseDuration
+```toml
+LeaseDuration = '0s' # Default
+```
+LeaseDuration is the maximum duration that the selected "best" node (as defined by SelectionMode) will be used,
```

Comment:
```suggestion
LeaseDuration is the minimum duration that the selected "best" node (as defined by SelectionMode) will be used,
```

Reply:
Isn't it the maximum? `checkLease` is not tied to when the node was selected, so we could have this timeline:

1. NodePool starts with `LeaseDuration='10m'`, lease timer start with it
2.  After 5m  an RPC goes down and a new "best" node is selected for a subscription
3.  1m later the better node comes back online 
4.  after 4m the lease timer triggers the lease check and switches to the better node

So in total, the node was up only 5m

Reply:
It's not difficult, the pool already keeps track of the active node inside the `selectNode()` method, so we can reset the timer there if a new node is selected, my only concern is that if the "best" node is flipping in and out of `NodeStateAlive` faster than `LeaseDuration ` then  the lease check might not happen at all

Reply:
Added in [`dc2bd13` (#10726)](https://github.com/smartcontractkit/chainlink/pull/10726/commits/dc2bd1381b1256c24934485d226010cc8d2f8cda)

Reply:
Updated in [`2416b9a` (#10726)](https://github.com/smartcontractkit/chainlink/pull/10726/commits/2416b9a6220a68de58935b488fa7486544a56a21)